Backoffice/Form/Type/ContentType.php

```diff
@@ -112,6 +114,8 @@ public function configureOptions(OptionsResolver $resolver)
             'is_statusable' => false,
             'need_link_to_site_defintion' => false,
             'delete_button' => false,
+            'delete_business_rules' => false,
```

Comment:
It sounds weird to bind an option with the button and the other with the manager. It should be show_delete_button and enable_delete_button or delete_right_rules and delete_business_rules.